### PR TITLE
SOLENG 9328

### PR DESCRIPTION
**Pull request by jonathankiddgeotab**
Added the Add-in Icon Validator (https://jira.geotab.com/browse/SOLENG-9328) to the bottom of the Developing Add-ins page.

### CODE REVIEW

File: src/software/guides/addin-icon-validator.html [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+                    <input type="radio" name="displayed" class="" id="neutral" checked onChange="handleFiles()">
+                    <label class="" for="neutral">Neutral</label>
+                    <input type="radio" name="displayed" id="selected" class="" onChange="handleFiles()">
+                    <label class="" for="selected">Selected</label>
+                </div>
+
+            </div>
+
+        </form>
+
```
Comment (muyuncaigeotab) [inline src/software/guides/addin-icon-validator.html:185]: Could you double-check if there's a missing opening \<form\> tag?

File: src/software/guides/addin-icon-validator.html [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+                        }
+
+                        if(errors == ""){
+                            errors += "There are no issues with the uploaded file. <br>";
+                        }
+
+                        text += "</p>";
+
+                        //PRINTING THE ERRORS ON THE SCREEN
+                        document.body.insertAdjacentHTML("beforeend", "<div class=\"container\" id=\"errors\"><h2>Errors:</h2>" + errors + "<br></div>");
```
Comment (muyuncaigeotab) [inline src/software/guides/addin-icon-validator.html:296]: Could we build a logic to not have the h2 element "Errors" when there are no issues with the uploaded file?

File: src/software/guides/addin-icon-validator.html (hunk 1 above)
Comment (jonathankiddgeotab) [reply src/software/guides/addin-icon-validator.html:185]: open form tag is on line 162 

<form name="uploadForm">

Comment (muyuncaigeotab) [reply src/software/guides/addin-icon-validator.html:185]: But you have another close form tag in line 323

Comment (jonathankiddgeotab) [reply src/software/guides/addin-icon-validator.html:185]: oh yes my mistake, just removed that it was not needed

File: src/software/guides/addin-icon-validator.html (hunk 2 above)
Comment (jonathankiddgeotab) [reply src/software/guides/addin-icon-validator.html]: added an if statement around the document.body.insertAdjacentHTML line to only display when errors != ""

**Pull request merged**